plugins/woocommerce/client/blocks/tests/e2e/tests/add-to-cart-with-options/add-to-cart-with-options.block_theme.spec.ts

```diff
@@ -134,7 +134,7 @@ test.describe( 'Add to Cart + Options Block', () => {
 			await addToCartButton.click();
 
 			await expect(
-				page.getByText( ' No matching variation found.' )
+				page.getByText( ' Please select product attributes before adding to cart.' )
```

Comment:
Could we take the opportunity to remove the leading space here? I think that's an overlook. :bow: 

```suggestion
				page.getByText( 'Please select product attributes before adding to cart.' )
```

Reply:
Done